### DISCUSSION TITLE
Colored drop-shadow

### DISCUSSION THREAD

**Discussion by SadiinsoSnowfall** [tailwindlabs/tailwindcss#6419] [Ideas]
The `shadow` utility supports custom color via the `shadow-color` utility.
For example `shadow-md shadow-red-500` will add a red shadow to the target element.
However this is not compatible with the `drop-shadow` utility: `drop-shadow-md shadow-red-500` will display a gray drop-shadow.

Would it be possible to make the `drop-shadow` utility use the `--tw-shadow-color` variable to accept custom colors ?

**Comment by SylvianHemus** [comment]
NEEDED
